js/util/helpers.js

```diff
@@ -882,6 +882,55 @@ define(["dao", "globals", "lib/knockout", "util/eventLog"], function (dao, g, ko
         return (arg > 0 ? "+" : "") + round(arg, d);
     }
 
+    /**
+     * Constructor for a sorter object. Use the objects sortF member as the sort
+     * function for the object list.
+     *
+     * @param {Array of String fields} sortBy sorting fields in order of application
+     */
+    function MultiSort(sortBy) {
```

Comment:
Couldn't this be a normal function that returns the sortF function? If so, I'd prefer that to a constructor function.